README.md

```diff
@@ -204,6 +204,26 @@ export PINECONE_ENV="Your pinecone region" # something like: us-east4-gcp
 
 ```
 
+## Weaviate Setup
+
+[Weaviate](https://weaviate.io/) is an open-source vector database. It allows to store data objects and vector embeddings from ML-models and scales seamlessly to billion of data objects. [An instance of Weaviate can be created locally (using Docker), on Kubernetes or using Weaviate Cloud Services](https://weaviate.io/developers/weaviate/quickstart). 
+Although still experimental, [Embedded Weaviate](https://weaviate.io/developers/weaviate/installation/embedded) is supported which allows the Auto-GPT process itself to start a Weaviate instance. To enable it, set `USE_WEAVIATE_EMBEDDED` to `True`. 
+
+#### Setting up enviornment variables
```

Comment:
Followed the instructions and ... it did not work 😂

Because it is missing the weaviate client.

So, perhaps edit to say something like "First, run `pip install weaviate-client`?

Reply:
@hsm207  HA!  I had assumed `pip install -r requirements.txt` would have been run. But you raise a good point. Have added to README.